examples/svelte/ssr/src/lib/data.ts

```diff
@@ -1,13 +1,22 @@
 import type { Post } from './types'
+import type { RequestEvent } from '@sveltejs/kit'
 
-export const getPosts = async (limit: number) => {
-  const response = await fetch('https://jsonplaceholder.typicode.com/posts')
+export const getPosts = async (
+  limit: number,
+  customFetch: RequestEvent['fetch'] = fetch,
```

Comment:
```suggestion
  customFetch: typeof fetch = fetch,
```
I think SvelteKit's fetch function satisfies this type, so maybe it's more succinct to do this instead of adding an extra import?

On a side note, should any of the documentation be updated as well?

Reply:
Thanks, I've fixed the type and updated the docs :)